Assets/Scripts/BlockScripts/Letter.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Letter 
+{
+    public string letter = string.Empty;
+    public int engWeight = 0;
+    public int frWeight = 0;
+
+    public void defineLetter(string n, int ew, int fw)
```

Comment:
ew and fw could be confusing, I'd leave a comment or change it to 'weight' if you go with the above comment

Reply:
Changed back to one weight.